target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-4i-edge-200.dts

```diff
@@ -0,0 +1,218 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Endian 4i Edge 200 Board Description
+ * Note: Endian UTM Mini are hardware clone of Endian Edge 200
+ * Copyright 2021 Pawel Dembicki <paweldembicki@gmail.com>
+ */
+
+/dts-v1/;
+
+#include "kirkwood.dtsi"
+#include "kirkwood-6281.dtsi"
+
+/ {
+	model = "Endian 4i Edge 200";
+	compatible = "endian,4i-edge-200", "marvell,kirkwood-88f6281", "marvell,kirkwood";
+
+	memory {
+		device_type = "memory";
+		reg = <0x00000000 0x20000000>;
+	};
+
+	aliases {
+		led-boot = &led_status_green;
+		led-failsafe = &led_status_orange;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_orange;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+		stdout-path = &uart0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-0 = <&pmx_led49 &pmx_led35 &pmx_led34>;
+		pinctrl-names = "default";
+
+		led_status_green: status_green {
+			label = "green:status";
+			gpios = <&gpio1 17 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_status_orange: status_orange {
+			label = "orange:status";
+			gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
+		};
+
+		sdcard {
+			label = "orange:sdcard";
+			gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "mmc0";
+		};
+	};
+};
+
+&eth0 {
+	status = "okay";
+	ethernet0-port@0 {
+		speed = <1000>;
+		duplex = <1>;
+	};
+};
+
+&eth1 {
+	status = "okay";
+	ethernet1-port@0 {
+		phy-handle = <&ethphyb>;
+	};
+};
+
+&mdio {
+	status = "okay";
+
+	ethphyb: ethernet-phy@b {
+		reg = <0x0b>;
+
+		marvell,reg-init =
+			/* link-activity, bi-color mode 4 */
+			<3 0x10 0xfff0 0xf>; /* Reg 3,16 <- 0xzzzf */
+	};
+
+	switch0: switch@11 {
+		compatible = "marvell,mv88e6085";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x11>;
+
+		ports {
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			port@0 {
+				reg = <0>;
+				label = "port1";
+			};
+
+			port@1 {
+			       reg = <1>;
+			       label = "port2";
+			};
+
+			port@2 {
+			       reg = <2>;
+			       label = "port3";
+			};
+
+			port@3 {
+			       reg = <3>;
+			       label = "port4";
+			};
+
+			port@5 {
+				reg = <5>;
+				label = "cpu";
+				phy-mode = "rgmii-id";
+				ethernet = <&eth0port>;
+				fixed-link {
```

Comment:
Please add a blank line above this line.

Reply:
Done.